OWNERS_ALIASES

```diff
@@ -390,3 +390,17 @@ aliases:
   - pmtk
   - eggfoobar
   - pliurh
+edge-enablement-reviewers:
```

Comment:
Indentation seems to be off: 

<img width="312" alt="Screenshot 2024-01-29 at 09 31 14" src="https://github.com/openshift/release/assets/5543238/a0ea1507-8b2d-4456-8641-1e051122d109">

Reply:
Is there a reason why Bulat is not in the reviewers?

Reply:
No reason other than I just missed it. I fixed the spacing and added Bulat to reviewers